### DISCUSSION TITLE
我的 2022 年总结 · posts/review-2022/

### DISCUSSION THREAD

**Discussion by giscus** [tcitry/tcitry.github.io#142] [Announcements]
# 我的 2022 年总结

转眼又到了 2022 年底，时间过得飞快，今年算是完成了几件有意义的事情，但这期间也充满了迷茫与艰难选择，内心发生了跌宕起伏的情绪变化，无论结果如何，2022 年发生的一切对我来说都是宝贵的经历。 工作 # 换了一份工

https://yindongliang.com/posts/review-2022/

**Comment by schecterdamien** [comment]
“于是在四月某一个不想上班的早晨，我给自己算了一卦” 😂
